pp/integrations/plugin-packs/procedures/hardware-servers-hp-ilo-xmlapi.md

```diff
@@ -2,34 +2,294 @@
 id: hardware-servers-hp-ilo-xmlapi
 title: HP Ilo XMLAPI
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **HP Ilo XMLAPI** brings a host template:
+
+* **HW-Server-Hp-Ilo-Xmlapi-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="HW-Server-Hp-Ilo-Xmlapi-custom" label="HW-Server-Hp-Ilo-Xmlapi-custom">
+
+| Service Alias   | Service Template                        | Service Description |
+|:----------------|:----------------------------------------|:--------------------|
+| Hardware-Global | HW-Hp-Ilo-Hardware-Global-Xmlapi-custom | Check components    |
+
+> The services listed above are created automatically when the **HW-Server-Hp-Ilo-Xmlapi-custom** host template is used.
+
+</TabItem>
+</Tabs>
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="Hardware-Global" label="Hardware-Global">
+
+The list of metrics can vary from one model to another.
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
```

Comment:
what about this sentence in the old doc? "A reading account is needed on HP ILO card."